invisible_cities/cities/components.py

```diff
@@ -1514,7 +1515,11 @@ def threshold_hits(hitc : HitCollection) -> HitCollection:
 
 
 @check_annotations
-def hits_corrector(map_fname : str, apply_temp : bool) -> Callable:
+def hits_corrector( filename   : str
+                  , apply_temp : bool
+                  , norm_strat : NormStrategy
+                  , norm_value : Optional[float] = 0
+                  ) -> Callable:
```

Comment:
Is doesn't appear that there's any testing for this function in `components_test.py`, is it worth doing so?

Reply:
Always!